### PR TITLE
fix: fix style error

### LINKED ISSUE #39288: FloatButton style error when custom size
### Reproduction link

[Edit on CodeSandbox](https://codesandbox.io/s/hidden-lake-mhzyu5?file=/src/App.tsx)

### Steps to reproduce

https://codesandbox.io/s/hidden-lake-mhzyu5?file=/src/App.tsx

### What is expected?

body居中

### What is actually happening?

body未居中

| Environment | Info |
| --- | --- |
| antd | 5.0.4 |
| React | 18.2.0 |
| System | mac |
| Browser | Chrome |

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [x] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

- close #39288 

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | fix: fix style error |
| 🇨🇳 Chinese | fix: 修复了按钮尺寸自定义时，内层body元素没有居中的问题 |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/float-button/style/index.tsx [hunk 1]
```diff
@@ -212,8 +212,9 @@ const sharedFloatButtonStyle: GenerateStyle<FloatButtonToken, CSSObject> = (toke
       },
 
       [`${componentCls}-body`]: {
-        width: floatButtonSize,
-        height: floatButtonSize,
+        margin: 'auto',
```
Comment (MadCcc) [inline components/float-button/style/index.tsx:215]: 这个是不是可以不要，style 是给到这个元素上的么？

**Comment by MadCcc** [comment]
记得好像有个 squre 的外层圆角改成 LG 的，一起改了吧

File: components/float-button/style/index.tsx (hunk 1 above)
Comment (li-jia-nan) [reply components/float-button/style/index.tsx:215]: style 是给到外层的，但是里面的宽高是写死的，外层尺寸发生变化了，里面不会跟着变化

Comment (MadCcc) [reply components/float-button/style/index.tsx:215]: 里面 100% 的话还需要 margin 吗

Comment (li-jia-nan) [reply components/float-button/style/index.tsx]: 是不用加，我去掉了

**Comment by li-jia-nan** [comment]
> 记得好像有个 squre 的外层圆角改成 LG 的，一起改了吧

done

**Comment by afc163** [comment]
留个重现链接？

**Comment by li-jia-nan** [comment]
> 留个重现链接？

- #39288

**Review by MadCcc — APPROVED** [review]

**Pull request merged**